### PR TITLE
Fixes to net-shaper and net.sh

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
`net-shaper` is able to partition the network. But on recovery it's unable to remove filters from testnet

#### Summary of Changes
Added a default filter, that allows packets with TOS 0 to go through. This fixes the SSH/SCP connectivity problem with the network nodes.

### CODE REVIEW

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

File: net/net.sh [hunk 1]
```diff
@@ -88,10 +88,6 @@ Operate a configured testnet
 
    --use-move                         - Build the move-loader-program and add it to the cluster
 
-   --operating-mode development|softlaunch
```
Comment (danpaul000) [inline net/net.sh]: @pgarg66 why did you remove this?  I implemented this to enable integration testing of a soft-launch genesis block.

Comment (CriesofCarrots) [reply net/net.sh]: Rebase issue, maybe? I think we need to preserve the `faucet-lamports` changes too.

Comment (danpaul000) [reply net/net.sh]: Yeah makes sense.  I'll PR a rollback
